Upstream-Status: Pending

Fix libacpi for x32
    
    libacpi doesn't use base_libdir for install.  This patch adds
    libdir support so that x32 libraries are installed properly.

Patch Received from:  H.J. Lu <hjl.tools@gmail.com>
Updated by: Ricardo Salveti <ricardo@opensourcefoundries.com> 2018/02/03

Signed-off-by: Nitin A Kamble <nitin.a.kamble@intel.com> 2011/12/05

Index: libacpi-0.2/Makefile
===================================================================
--- libacpi-0.2.orig/Makefile
+++ libacpi-0.2/Makefile
@@ -42,13 +42,13 @@ install: all
 	@mkdir -p ${DESTDIR}${PREFIX}/include
 	@cp -f libacpi.h ${DESTDIR}${PREFIX}/include
 	@chmod 644 ${DESTDIR}${PREFIX}/include/libacpi.h
-	@echo installing library to ${DESTDIR}${PREFIX}/lib
-	@mkdir -p ${DESTDIR}${PREFIX}/lib
-	@cp -f libacpi.a ${DESTDIR}${PREFIX}/lib
-	@chmod 644 ${DESTDIR}${PREFIX}/lib/libacpi.a
-	@cp -f ${SONAME} ${DESTDIR}${PREFIX}/lib/
-	@chmod 644 ${DESTDIR}${PREFIX}/lib/${SONAME}
-	@ln -s ${SONAME} ${DESTDIR}${PREFIX}/lib/libacpi.so
+	@echo installing library to ${DESTDIR}${LIBDIR}
+	@mkdir -p ${DESTDIR}${LIBDIR}
+	@cp -f libacpi.a ${DESTDIR}${LIBDIR}
+	@chmod 644 ${DESTDIR}${LIBDIR}/libacpi.a
+	@cp -f ${SONAME} ${DESTDIR}${LIBDIR}/
+	@chmod 644 ${DESTDIR}${LIBDIR}/${SONAME}
+	@ln -s ${SONAME} ${DESTDIR}${LIBDIR}/libacpi.so
 	@echo installing test-libacpi to ${DESTDIR}${PREFIX}/bin
 	@mkdir -p ${DESTDIR}${PREFIX}/bin
 	@cp -f test-libacpi ${DESTDIR}${PREFIX}/bin
